shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
I tried giving `main-content` a `z-index` of -1, but then it didn't accept clicks at all. I don't fully understand what was happening, but it looked like the outer `<body>` tag was in front of it.